### PR TITLE
Satellite Charts

### PR DESCRIPTION
**Pull request by urchykli**
The three charts are complete, but I wasn't sure about the dates for the Long March Launches chart. Should I show all years on the x axis like in the google sheet or leave it as is? Thanks!

### CODE REVIEW

**Comment by urchykli** [comment]
All four charts are now included. Thanks!

File: satellites/launches/js/bar.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    title: {
+      text: "Number of Launches"
+    }
+    // labels: {
+    //   format: "${value}"
+    // }
+  },
+  // Tooltip
+  tooltip: {
+    valueSuffix: " Cinema Screens",
```
Comment (jnschrag) [inline satellites/launches/js/bar.js]: This should probably be removed :)

File: satellites/payloads/js/lines.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  },
+  // Y Axis
+  yAxis: {
+    title: {
+      text: "Number of Payloads Per Launch"
+    }
+  },
+  // Tooltip
+  tooltip: {
+    valueSuffix: " Cinema Screens",
```
Comment (jnschrag) [inline satellites/payloads/js/lines.js]: This should be removed.

File: satellites/payloads/js/lines.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  data: {
+    googleSpreadsheetKey: "1lB45XQ9UxUY_951-b0RSGGmyGkVX9JshmmkrfKAL7e0",
+    googleSpreadsheetWorksheet: 4
+  },
+  // General Chart Options
+  chart: {
+    type: "line"
+  },
+  // Colors
+  colors: ['#ed392a'],
```
Comment (jnschrag) [inline satellites/payloads/js/lines.js]: No need to explicitly define colors here. It'll pull them automatically in order, so red would be the first one called.

**Comment by urchykli** [comment]
I've made the following updates.
- Shared tooltip for Launches
- Removed Cinema Screens from all charts
- Link to remote Highcharts theme
- Remove unnecessary color from Payloads
- Removed "year" from all charts
- Update color pallet for pie chart
- Rename directory to Space Capabilities

Thanks!

**Comment by urchykli** [comment]
Per request from the program, I've made the following changes.
- Updated theme on china-space-timeline/humans
- Updated tooltip on Launches chart
- Changed purple to green on the Launches chart
- Made "per" lowercase on the Payloads chart

Thanks!

**Pull request merged**